backend/lcfs/db/migrations/versions/2024-11-27-20-48_b69a33bbd135.py

```diff
@@ -0,0 +1,34 @@
+"""Add notifications email to user_profile
+
+Revision ID: b69a33bbd135
+Revises: 0775a141d335
+Create Date: 2024-11-27 20:48:24.724112
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "b69a33bbd135"
+down_revision = "0775a141d335"
```

Comment:
Need to update this to the previous revision 4038ff8d8c49

Reply:
Thanks, Arturo, for reviewing my PR. I’ve resolved the issue!